i18n/locales/de.json

```diff
@@ -12,5 +12,11 @@
   "selectedDate": "Gewähltes Datum",
   "waterLevelOfLake": "Wasserpegel des Sees",
   "waterLevel": "Wasserpegel",
-  "notAvailable": "Nicht verfügbar"
+  "notAvailable": "Nicht verfügbar",
+  "loadingText": "Laden der Daten, bitte warten...",
+  "chartX": "Datum",
+  "chartY": "Tiefe (m)",
+  "chartTitle": "Tiefe für",
+  "chartLegend": "Tiefe im Zeitverlauf",
+  "chartEmpty": "Es sind keine Tiefenangaben verfügbar für"
```

Comment:
@alifarooq93 The translation does not fit to the keys. So please adjust the keys so that it is clear that they are lake specific